web/templates/settings/notification_settings.hbs

```diff
@@ -65,15 +65,19 @@
         </div>
 
         <div class="input-group">
-            <label for="automatically_follow_topics_policy" class="settings-field-label">{{ settings_label.automatically_follow_topics_policy }}</label>
+            <label for="automatically_follow_topics_policy" class="settings-field-label">{{ settings_label.automatically_follow_topics_policy }}
```

Comment:
It would look nicer to have both the settings label on its own line, indented, just like the help link widget on the line below.

Reply:
Thanks Resolved in new commit.

Reply:
I think you failed to push the update properly?